### PR TITLE
Added vulnerable cookie endpoints and enabled tests.

### PR DESCRIPTION
**Pull request by DDJavierSantos**
## Description

Testing cookie vulnerabilities for No SameSite, No HttpOnly and insecure cookie

## Motivation

to cover all of the implemented cookie vulnerabilites in java

## Reviewer checklist

* [ ] If this PR modifies anything else than strictly the default scenario, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/system-tests-ci.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

### CODE REVIEW

File: tests/appsec/iast/sink/test_insecure_cookie.py [hunk 1]
```diff
@@ -11,8 +11,20 @@
 
 
 @coverage.basic
-@released(dotnet="?", java="?", golang="?", php_appsec="?", python="?", ruby="?")
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?")
 @released(nodejs={"express4": "4.1.0", "*": "?"})
+@released(
```
Comment (smola) [inline tests/appsec/iast/sink/test_insecure_cookie.py:16]: When the release version is the same, it's probably better to just do `java="1.18.0"` and then use additional `@missing_feature(weblog_variant="akka-http", reason="Missing endpoint")` to exclude some variants.

Comment (cbeauchesne) [reply tests/appsec/iast/sink/test_insecure_cookie.py:16]: Indeed, and you can even do : 

```python
@released(java={"akka-http": "?", "*": "1.18.0"})
```

**Review by cbeauchesne — APPROVED** [review]

File: tests/appsec/iast/sink/test_sql_injection.py [hunk 2]
```diff
@@ -26,7 +26,6 @@
         "resteasy-netty3": "1.11.0",
         "jersey-grizzly2": "1.11.0",
         "vertx3": "1.12.0",
-        "akka-http": "1.12.0",
```
Comment (smola) [inline tests/appsec/iast/sink/test_sql_injection.py:29]: Why? I think this is an error (disabling this for SQL Injection).

File: utils/build/docker/java/jersey-grizzly2/src/main/java/com/datadoghq/jersey/IastSinkResource.java [hunk 3]
```diff
@@ -188,4 +188,57 @@ public Response secureUnvalidatedRedirect() throws URISyntaxException {
     public Response insecureUnvalidatedRedirect(@FormParam("location") final String location) throws URISyntaxException {
         return Response.status(Response.Status.TEMPORARY_REDIRECT).location(new URI(location)).build();
     }
+
+    @GET
+    @Path("/insecure-cookie/test_empty_cookie")
+    public Response insecureCookieEmptyCookie() {
+        return Response.status(Response.Status.OK).header("Set-Cookie", "").build();
+    }
+    @GET
```
Comment (smola) [inline utils/build/docker/java/jersey-grizzly2/src/main/java/com/datadoghq/jersey/IastSinkResource.java:197]:
```suggestion

    @GET
```

File: utils/build/docker/java/jersey-grizzly2/src/main/java/com/datadoghq/jersey/IastSinkResource.java [hunk 4]
```diff
@@ -188,4 +188,57 @@ public Response secureUnvalidatedRedirect() throws URISyntaxException {
...
+    public Response  noSameSiteCookieSecure() {
+        return Response.status(Response.Status.OK).header("Set-Cookie", "user-id=7;Secure;HttpOnly=true;SameSite=Strict").build();
+    }
+
+    @GET
+    @Path("/no-httponly-cookie-cookie/test_empty_cookie")
+    public Response  noHttpOnlyCookieEmptyCookie() {
+        return Response.status(Response.Status.OK).header("Set-Cookie", "").build();
+    }
+    @GET
```
Comment (smola) [inline utils/build/docker/java/jersey-grizzly2/src/main/java/com/datadoghq/jersey/IastSinkResource.java:232]:
```suggestion

    @GET
```

File: utils/build/docker/java/resteasy-netty3/src/main/java/com/datadoghq/resteasy/IastSinkResource.java [hunk 5]
```diff
@@ -183,4 +183,55 @@ public Response secureUnvalidatedRedirect() throws URISyntaxException {
     public Response insecureUnvalidatedRedirect(@FormParam("location") final String location) throws URISyntaxException {
         return Response.status(Response.Status.TEMPORARY_REDIRECT).location(new URI(location)).build();
     }
+    @GET
```
Comment (smola) [inline utils/build/docker/java/resteasy-netty3/src/main/java/com/datadoghq/resteasy/IastSinkResource.java:186]:
```suggestion

    @GET
```

File: utils/build/docker/java/resteasy-netty3/src/main/java/com/datadoghq/resteasy/IastSinkResource.java [hunk 6]
```diff
@@ -183,4 +183,55 @@ public Response secureUnvalidatedRedirect() throws URISyntaxException {
...
+    @Path("/insecure-cookie/test_empty_cookie")
+    public Response insecureCookieEmptyCookie() {
+        return Response.status(Response.Status.OK).header("Set-Cookie", "").build();
+    }
+    @GET
```
Comment (smola) [inline utils/build/docker/java/resteasy-netty3/src/main/java/com/datadoghq/resteasy/IastSinkResource.java:191]:
```suggestion

    @GET
```

File: utils/build/docker/java/vertx3/src/main/java/com/datadoghq/vertx3/Main.java [hunk 7]
```diff
@@ -148,6 +148,7 @@ public static void main(String[] args) {
                     ctx.response().end("ok");
                 });
 
+
```
Comment (smola) [inline utils/build/docker/java/vertx3/src/main/java/com/datadoghq/vertx3/Main.java:151]:
```suggestion
```

Change not needed.

**Review by smola — CHANGES_REQUESTED** [review]

File: tests/appsec/iast/sink/test_sql_injection.py (hunk 2 above)
Comment (DDJavierSantos) [reply tests/appsec/iast/sink/test_sql_injection.py:29]: yes, error

Comment (DDJavierSantos) [reply tests/appsec/iast/sink/test_sql_injection.py]: fixed

**Review by smola — APPROVED** [review]

**Pull request merged**